components/src/atoms/buttons/EmptySelectorButton.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react'
+import { Flex } from '../../primitives'
+import {
+  BORDERS,
+  COLORS,
+  Icon,
+  SPACING,
+  StyledText,
+  Btn,
+  JUSTIFY_CENTER,
+  JUSTIFY_START,
+  ALIGN_CENTER,
+  FLEX_MAX_CONTENT,
+} from '../..'
+import type { IconName } from '../..'
+
+interface EmptySelectorButtonProps {
+  onClick: () => void
+  text: string
+  size: 'large' | 'small'
```

Comment:
nit
If the design team knows which size will be used more often, making `size` optional and setting the default value may be good.

```suggestion
  size?: 'large' | 'small'
  
  // then
  {..., size = 'large' } = props
```